lib/Onyx.js

```diff
@@ -424,7 +472,7 @@ function keysChanged(collectionKey, partialCollection) {
                 subscriber.withOnyxInstance.setState((prevState) => {
                     const data = cachedCollection[subscriber.key];
                     const previousData = prevState[subscriber.statePropertyName];
-                    if (data === previousData) {
+                    if (deepEqual(data, previousData)) {
```

Comment:
Same here. Is this related or just something that seems more correct?

Reply:
Yeah again, you are right, thats just something thats belonging more to a different set of performance improvements. Will remove it from here and do a follow up issue/PR.